src/site/markdown/server/manage-webadmin.md

```diff
@@ -2496,6 +2497,45 @@ Response codes:
 
 Not implemented yet.
 
+## Deleted Messages Vault
+
+Deleted messages from an user will be stored in a restricted place where only administrators can access.
+This place is called the Deleted Messages Vault.
+
+ - [Restore Deleted Messages](#Restore_deleted_messages)
+
```

Comment:
We need to add a discalaimer: this is only supported by cassandra guice products...